packages/app/components/Nav/ProfileDrawer.tsx

```diff
@@ -24,35 +24,34 @@ const AvatarButton = styled.div`
   cursor: pointer;
 `;
 
-export default function Settings(): ReactElement {
+interface ProfileDrawerProps {
+  courseId: number;
+}
+
+export default function ProfileDrawer({
+  courseId,
+}: ProfileDrawerProps): ReactElement {
   const profile = useProfile();
-  const [isNotifOpen, setIsNotifOpen] = useState(false);
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
   const loginPath = isProd() ? "/login" : "/dev";
 
   return (
     <>
-      <NotificationSettingsModal
-        visible={isNotifOpen}
-        onClose={() => setIsNotifOpen(false)}
-      />
       <NoPaddingPopover
         content={
           isPopoverOpen && (
             <Menu mode="inline">
               <Menu.ItemGroup title="Settings">
-                {/* Uncomment when we implement profile settings */}
-                {/* <Menu.Item icon={<UserOutlined />}>
-                Profile Settings
-              </Menu.Item> */}
                 <Menu.Item
-                  onClick={() => {
-                    setIsPopoverOpen(false);
-                    setIsNotifOpen(true);
-                  }}
-                  icon={<SettingOutlined />}
+                  icon={
+                    <SettingOutlined /> /* Food for thought, do we want to turn this into a profile outline or settings outline*/
+                  }
```

Comment:
Minor thing but can we remove the Settings `Menu.ItemGroup` since there is only going to be a single `Menu.Item` inside of it?

Reply:
![image](https://user-images.githubusercontent.com/33675688/100391408-5428e980-3001-11eb-9d10-4422a920a2c9.png)
looks like this -- thoughts?

Reply:
Hmm looks less symmetrical now. We could probably remove the External Pages `Menu.ItemGroup` too?

Reply:
Oop that's what you did, thank you!